README.md

```diff
@@ -270,6 +272,20 @@ export SPHERE_CLIENT_SECRET="your-client-secret"
 #### ElasticIo
 _(Coming soon)_
 
+#### UserAgent
+A synchronous module that builds _user\_agent_ according to the standard specified [here](https://sphere.atlassian.net/wiki/x/7ACjB)
+
+sphere-node-sdk module must be installed in the node_modules because it's required to build the user_agent
```

Comment:
I just realised this is nasty and is only because we make a module of it. As in if it's part of a project it will have the SDK as a dependency itself.